parts/language/dict_en.lua

```diff
@@ -272,12 +272,12 @@ FNNS and     {"Support 3",
     {"TGM",
         "tetrisgrandmaster tetristhegrandmaster",
         "game",
-        "Arcade | Singleplayer/Local Multiplayer\nTetris The Grand Master, an arcade Tetris series. Titles like S13 and GM were from this series.\n\nTGM3 is the most well-known game in this series.",
+        "Arcade | Singleplayer/Local Multiplayer\nTetris The Grand Master, an arcade Tetris series. Titles like S13 and GM originate from this series.\n\nTGM3 is the most well-known game in this series.",
     },
     {"DTET",
         "dtet",
         "game",
-        "Windows | Singleplayer\nA game based on TGM’s Classic rule with 20G and a powerful rotation system. Decent controls but has no customization other than control mappings. The game is a bit hard to find now, and you may need to install the required DLL files manually.",
+        "Windows | Singleplayer\nA game based on TGM's Classic rule with 20G and a powerful rotation system, with decent controls but no customization other than control mappings. The game is a bit hard to find now, and you may have to install the necessary DLL files manually.",
     },
```

Comment:
Can we add this link for how to install DTET? "https://t-sp.in/dtet"

Reply:
Added.